web/src/components/layout/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+export * from "./Layout";
+export { default as Layout } from "./Layout";
+export { default as Center } from "./Center";
+export { default as DBusError } from "./DBusError";
+export { default as LoadingEnvironment } from "./LoadingEnvironment";
```

Comment:
I think this is PR is a good opportunity for making `LoadingEnvironment` a more generic `Loading` component. In fact, it already receives a[ _text_](https://github.com/yast/d-installer/blob/2de401429030d7b22febc58aad4758910aabe303/web/src/LoadingEnvironment.jsx#L29) prop that can be accordingly adapted too. What do you think?